build/common.sh

```diff
@@ -682,6 +682,7 @@ function kube::build::sync_to_container() {
   # they will not be re-generated by 'make'.
   kube::build::rsync \
     --delete \
+    --filter='- /.git/' \
```

Comment:
If we're re-adding this, may as well prevent generated files from being rsync'd in too. My original patch:
```
diff --git a/build/common.sh b/build/common.sh
index f6def8f3e3bae..1223510dd0410 100755
--- a/build/common.sh
+++ b/build/common.sh
@@ -682,14 +682,10 @@ function kube::build::sync_to_container() {
   # they will not be re-generated by 'make'.
   kube::build::rsync \
     --delete \
-    --filter='+ /staging/**' \
-    --filter='- /.git/' \
     --filter='- /.make/' \
     --filter='- /_tmp/' \
     --filter='- /_output/' \
     --filter='- /' \
-    --filter='- zz_generated.*' \
-    --filter='- generated.proto' \
     "${KUBE_ROOT}/" "rsync://k8s@${KUBE_RSYNC_ADDR}/k8s/"
```